clang/lib/Headers/openmp_wrappers/complex_cmath.h

```diff
@@ -64,8 +64,13 @@ template <class _Tp> __DEVICE__ _Tp norm(const std::complex<_Tp> &__c) {
 }
 
 // conj
-
-template <class _Tp> std::complex<_Tp> conj(const std::complex<_Tp> &__c) {
+#ifdef _GLIBCXX20_CONSTEXPR
+#define CXX20_CONSTEXPR_DEVICE __DEVICE__
```

Comment:
I wonder why this is not `constexpr` in C++20 but `__DEVICE__` instead?

Reply:
@shiltian I followed the existing approach used to define constexpr for handling OpenMP offloading in this wrapper. The __DEVICE__ macro indeed looks better suited with GPU offloading environments.